### ISSUE TITLE
Rename Generic Functions in lib.php

### ISSUE THREAD

**Issue by eduardokraus** [marcusgreen/moodle-tool_driprelease#10]
There are some functions in lib.php with very generic names that may cause conflicts with other plugins. For example, the functions "row_fill" and "get_table_data".

**Comment by marcusgreen** [comment]
I agree, and so I am working on a refactor to use a class called driprelease, which means I am converting existing references to a class called driprelease to be named dripdata, which is a better name.
The branch is here, and I will merge into main once I am happy it all works. I am currently getting a behat failure.

**Comment by marcusgreen** [comment]
I have now completed the refactoring and merged into the main branch.
